WordPress-Core/ruleset.xml

```diff
@@ -150,6 +150,9 @@
 		<!-- Covers rule: When type casting, do it like so: $foo = (boolean) $bar; -->
 		<rule ref="Generic.Formatting.SpaceAfterCast"/>
 		<rule ref="WordPress.WhiteSpace.CastStructureSpacing"/>
+		<rule ref="WordPress.WhiteSpace.CastStructureSpacing.ContainsWhiteSpace">
+			<type>warning</type>
```

Comment:
Oh and another thing: If it would be decided that this should be an `error`, I would suggest to change the other two checks in this sniff to `error`s as well.